content/_layouts/post.html.haml

```diff
@@ -4,20 +4,10 @@ section: blog
 ---
 
 :css
-  div.content {
-    position: relative;
-    overflow: hidden;
-    width: 100%;
-    padding-top: 56.25%; 
-  }
-  iframe {
-    position: absolute;
-    top: 0;
-    left: 0;
-    bottom: 0;
-    right: 0;
-    width: 100%;
-    height: 100%;
+  div.content iframe {
```

Comment:
```suggestion
  .videoblock iframe {
```
I think this makes it more clear that it's about videos.

Reply:
@zbynek Yeah! Thank you for your suggestions